packages/@react-stately/selection/src/SelectionManager.ts

```diff
@@ -373,8 +385,10 @@ export class SelectionManager implements MultipleSelectionManager {
       }
     } else if (e && e.shiftKey) {
       this.extendSelection(key);
-    } else {
+    } else if (this.selectionBehavior === 'toggle' || (e && e.metaKey)) {
       this.toggleSelection(key);
+    } else {
+      this.replaceSelection(key);
```

Comment:
Kinda awkward trying to select additional individual rows in a table using the meta key on mac cuz CMD+Space brings up spotlight search heh. CMD+Enter works fine though so maybe this is ok...